src/components/Drawer/index.js

```diff
@@ -0,0 +1,99 @@
+import { Component } from 'pet-dex-utilities';
+import { listenBreakpoint } from '../../utils/breakpoints/breakpoints';
+import { makeSwipable } from '../../utils/swiper';
+import close from './images/close.svg';
+import line from './images/line.svg';
+import './index.scss';
+
+const events = ['open', 'close', 'title:change', 'content:change'];
+
+const html = `
+  <div class="drawer" data-select="drawer">
+    <div class="drawer__wrapper" data-select="drawer-wrapper">
+      <div class="drawer__nav">
+        <span class="drawer__title" data-select="title"></span>
+        <button class="drawer__close" data-select="close">
+          <img class="drawer__close--icon" src="${close}">
+          <img class="drawer__close--line" src="${line}">
+        </button>
+      </div>
+      <div class="drawer__content" data-select="content"></div>
+    </div>
+  </div>
+`;
+
+export default function Drawer({ title, content }) {
+  Component.call(this, { html, events });
+
+  const drawer = this.selected.get('drawer');
+  const drawerWrapper = this.selected.get('drawer-wrapper');
+  this.isOpen = false;
+
+  this.selected.get('close').addEventListener('click', () => this.close());
+
+  this.setTitle(title);
+  this.setContent(content);
+  makeSwipable(drawerWrapper);
```

Comment:
Não é essa a funçào pra usar no swipe.. tem q ser um evento padrào mesmo... Ou to errado, lembro de falarmos sobre isso

Reply:
Tem que usar essa mesmo, confirmei com o djogo

Reply:
Porque?

Porque apenas o evento não funciona?